lib/FusionInventory/Agent/SNMP/MibSupport/Hwg.pm

```diff
@@ -0,0 +1,64 @@
+package FusionInventory::Agent::SNMP::MibSupport::Hwg;
+
+use strict;
+use warnings;
+
+use parent 'FusionInventory::Agent::SNMP::MibSupportTemplate';
+
+use FusionInventory::Agent::Tools;
+use FusionInventory::Agent::Tools::SNMP;
+
+# See Hwg-MIB
+
+use constant hwg        => '.1.3.6.1.4.1.21796';
+use constant hwgModel   => '.1.3.6.1.2.1.1.1.0';
+use constant hwgWldMac  => hwg . '.4.5.70.1.0';
+use constant hwgSteMac  => hwg . '.4.1.70.1.0';
+
+our $mibSupport = [
+    {
+        name    => "hwg",
+        sysobjectid => getRegexpOidMatch(hwg)
+    }
+];
+
+sub getType {
+    return 'NETWORKING';
+}
+
+sub getManufacturer {
+    return 'HW group s.r.o';
+}
+
+sub getSerial {
+    my ($self) = @_;
+
+    my $serial = getCanonicalMacAddress($self->get(hwgWldMac) || $self->get(hwgSteMac));
```

Comment:
Using your walks to test the PR, it seems the MAc is wrongly read. This may work on live systems, but to be safe, can you update your code to do the following ?
```suggestion
    my $serial = getCanonicalMacAddress(getCanonicalString($self->get(hwgWldMac) || $self->get(hwgSteMac)));
```